app/js/lib/foxbox/settings.js

```diff
@@ -89,8 +87,10 @@ export default class Settings extends Model {
 
   set localOrigin(origin) {
     this._localOrigin = origin ?
-        (new URL(origin)).origin : DEFAULT_LOCAL_ORIGIN;
-    storage.setItem(`${PREFIX}localOrigin`, this._localOrigin);
+        (new URL(origin)).origin : undefined;
+    if (this._localOrigin) {
+      storage.setItem(`${PREFIX}localOrigin`, this._localOrigin);
+    }
```

Comment:
I'm not sure that I understand what you mean here.

Are you suggesting the following scenario?

1. Discover box with url X.
2. Start using box with url X.
3. Discover that box X changed to url Y (note that this requires resetting the box)
4. Fail to use Y connection
5. Go back to url X.